src/access/wac.ts

```diff
@@ -124,3 +140,27 @@ export async function getGroupAccess(
 ): Promise<Access | null> {
   return getActorAccess(resource, group, getGroupAccessWac, options);
 }
+
+/**
+ * For a given Resource, look up its metadata, and read the Access permissions
+ * granted to the given Group.
```

Comment:
(There's a couple more mentions of groups and even agents, but I presume you can find those yourself :) )

(But maybe proofread the group's docs as well.)

```suggestion
 * granted to everyone.
```

Reply:
Ah yeah sorry about that, I assumed that working on two parallel branches would be easier to merge, but conflict resolution was actually quite messy, I focused on the working code and forgot to check the comments ^^

Reply:
4f43596db728ac179e6e7e9d949702787b1bbab9